src/material/dialog/testing/dialog-harness.ts

```diff
@@ -63,4 +75,24 @@ export class MatDialogHarness extends ContentContainerComponentHarness<string> {
   async close(): Promise<void> {
     await (await this.host()).sendKeys(TestKey.ESCAPE);
   }
+
+  /** Gets te dialog's text. */
+  async getText() {
+    return (await this.host()).text();
+  }
+
+  /** Gets the dialog's title text. */
+  async getTitleText() {
+    return (await this._title())?.text() ?? '';
+  }
+
+  /** Gets the dialog's content text. */
+  async getContentText() {
```

Comment:
I'm not sure that this and `getActionsText` will be very useful since they can have all sorts of content. E.g. the actions usually contains one or more buttons.

Reply:
They're useful when people want to just make a quick test that a certain piece of content shows up in the dialog. I actually created this PR because I found a case where people were using a separate harness system and doing exactly these kind of tests